File: web/blocks/tags/unrestricted-tags.dart
// NetTango Copyright (C) Michael S. Horn, Uri Wilensky, and Corey Brady. https://github.com/NetTango/NetTango

part of NetTango;

class UnrestrictedTags extends ConcreteTags {
  ConcreteTags clone() {
    return this;
  }

  bool check(Iterable<Block> blocks) {
    return true;
  }
}

File: src/blocks/tags/unrestricted-tags.ts
// NetTango Copyright (C) Michael S. Horn, Uri Wilensky, and Corey Brady. https://github.com/NetTango/NetTango

import { Block } from "../block"
import { ConcreteTags } from "./concrete-tags"

class UnrestrictedTags extends ConcreteTags {
  clone(): ConcreteTags {
    return this
  }

  check(blocks: Block[]): boolean {
    return true
  }
}

export { UnrestrictedTags }